nethttp/client.go

```diff
@@ -242,8 +242,14 @@ func (h *Tracer) dnsStart(info httptrace.DNSStartInfo) {
 	)
 }
 
-func (h *Tracer) dnsDone(httptrace.DNSDoneInfo) {
+func (h *Tracer) dnsDone(info httptrace.DNSDoneInfo) {
 	h.sp.LogFields(log.String("event", "DNSDone"))
```

Comment:
I meant do a single span.LogFields() call with event=DNSDone, but append multiple `addr` fields to the slice.

If you're concerned with multiple fields having the same key (it's fine in Jaeger, but I guess the spec doesn't say exactly if it's allowed), then logging multiple events makes sense, but I would have `event=DNSDone` in all logs

Reply:
Ah sorry I misunderstood..  should be fixed now.